tests/test_robot.py

```diff
@@ -50,12 +51,13 @@
 from viam.services.motion.client import MotionClient
 from viam.utils import dict_to_struct, message_to_struct, struct_to_message
 
-from .mocks.components import MockArm, MockCamera, MockMotor, MockSensor
+from .mocks.components import MockArm, MockCamera, MockMotor, MockMovementSensor, MockSensor
 
 RESOURCE_NAMES = [
     ResourceName(namespace="rdk", type="component", subtype="arm", name="arm1"),
     ResourceName(namespace="rdk", type="component", subtype="camera", name="camera1"),
     ResourceName(namespace="rdk", type="component", subtype="motor", name="motor1"),
+    # ResourceName(namespace="rdk", type="component", subtype="sensor", name="movement_sensor1"),
```

Comment:
As a general policy, I think it's best to not have commented code committed. It is often unclear what purpose it serves/used to serve, and there's no validation that the code still works as expected. Probably best to get rid of this entirely.